app/mailers/user_mailer.rb

```diff
@@ -242,13 +242,15 @@ def add_email_associated_with_another_account
     end
   end
 
-  def account_verified(date_time:, sp_name:)
+  def account_verified(date_time:, profile:)
+    attachments.inline['verified.png'] = File.read('app/assets/images/email/user-signup-ial2.png')
     with_user_locale(user) do
+      @presenter = Idv::AccountVerifiedEmailPresenter.new(profile:)
+      @hide_title = true
       @date = I18n.l(date_time, format: :event_date)
-      @sp_name = sp_name
       mail(
         to: email_address.email,
-        subject: t('user_mailer.account_verified.subject', sp_name: @sp_name),
+        subject: t('user_mailer.account_verified.subject', app_name: APP_NAME),
```

Comment:
The email subject now indicates that the user has proofed with Login.gov, rather than with a given SP, and moves the SP info into the body of the email.